content/posts/ansible-install.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Install Ansible"
+date: 2021-03-1
+draft: false
+description: "Ansible Opensource automation platform"
+tags: ["opensource", "automation", "ssh", "installation"]
+author: "ShowMeYourSkil"
+---
+
+# Ansible
+
+Hey, in this articel I would like to tell you something about Ansible and how you can install it. 
+
+# What is ansible?
+
+Ansible is a automation platform to automate process. Example like install full automatic Virtual Machines oder Windows updates and more..
```

Comment:
processes. For example like installing

Reply:
I want add processes in my next articel... this articel is only how about install ansbile

Reply:
i tried to mark some spelling and grammar mistakes.....

Reply:
uhm no xD

I've give you write perms

Reply:
Please view my last commit. Bzw view this file now :D